backend/functions/lib/adapters/types/persistence.gateway.d.ts

```diff
@@ -1,7 +1,7 @@
 import { RecordObj, ObjectKeyVal } from './basic';
 import { DbResult } from './database';
 import { ApplicationContext } from './basic';
-import { AttorneyListDbResult } from './attorneys';
+import { AttorneyUser } from '../../../../../common/src/cams/users';
```

Comment:
I wasn't sure where this type came from when I was seeing it. I like this change!